packages/cli/src/runAndroid/runAndroid.js

```diff
@@ -215,9 +215,28 @@ function startServerInNewWindow(port, terminal = process.env.REACT_TERMINAL) {
   /**
    * Set up the `.packager.(env|bat)` file to ensure the packager starts on the right port.
    */
-  const launchPackagerScript = require.resolve(
-    `react-native/scripts/${scriptFile}`
-  );
+  let launchPackagerScript;
+  try {
+    launchPackagerScript = require.resolve(
+      `react-native/scripts/${scriptFile}`
+    );
+  } catch (e) {
+    /**
+     * If the `react-native` package is missing, check if we are running from source.
+     *
+     * From `node_modules/@react-native-community/cli/build/run-android to `scripts/`
+     */
+    const possibleScriptLocation = path.resolve(
+      __dirname,
+      `../../../../../scripts/${scriptFile}`
```

Comment:
or you could add a new flag `pathToReactNative` which you could pass to RNTester scripts?

Reply:
Yeah, I think this is a good idea and would solve few other issues we had already in `start` command. Let me rewrite it by adding global `--pathToReactNative` flag (similar to root) so that users can pass it and we can decide where to use it.

This is amazing for debugging too!